crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/BenefitsMapper.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
+
+import uk.gov.justice.laa.crime.applications.adaptor.enums.BenefitDetails;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.AssessmentDetail;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.general.Benefit;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class BenefitsMapper {
+
+    private static final String UNIVERSAL_CREDIT = "Universal Credit";
+    private static final String JSA = "JSA";
+    public List<AssessmentDetail> mapBenefits(List<Benefit> benefits) {
+        List<AssessmentDetail> assessmentDetails = new ArrayList<>();
+
+        if (Objects.nonNull(benefits)) {
+            for (Benefit benefit : benefits) {
+                AssessmentDetail assessmentDetail = new AssessmentDetail();
+                String benefitType = benefit.getType().value();
+                BenefitDetails benefitDetail = BenefitDetails.findByValue(benefitType);
```

Comment:
For example the expected exception is not handled here?

Reply:
As mentioned in previous comments, we never throw an exception.